### PR TITLE
Add `vmin` and `vmax` specification to `mne.Evoked.animate_topomap`

### LINKED ISSUE #10725: Specify vmin vmax color range in animated topomap
#### Describe the new feature or enhancement
Add the possibility to specify the value bounds of the colormap range in the _mne.Evoked.animate_topomap_ function.
The _mne.viz.plot_topomap_ function implements vmin and vmax for static topomaps but there seems to be no equivalent in the animated version of the function.

#### Describe your proposed implementation
 _mne.viz.plot_topomap_  function implements vmin and vmax parameters (lines 1700-1704 from topomap.py)
```
vmin = np.min(vlims)
    vmax = np.max(vlims)
    cmap = _setup_cmap(cmap, n_axes=n_times, norm=vmin >= 0)
```

animate_topomap could implement vmin and vmax parameters in the same way. They could be passed on from to mne.Evoked.animate_topomap to the params dictionnary (line 2361-2369 of topomap.py)
```
params = dict(data=data, pos=pos, all_times=evoked.times, frame=0,
                  frames=frames, butterfly=butterfly, blit=blit,
                  pause=False, times=times, time_unit=time_unit,
                  clip_origin=clip_origin,vmin=vmin,vmax=vmax)
[...]
 animate_func = partial(_animate, ax=ax, ax_line=ax_line, params=params)
```
as the __animate_ function already seems to allow vmin and vmax into params : (lines 2267-2268 of __animate_ function
    vmin = params['vmin']
    vmax = params['vmax']

Sorry if this isn't clear, this is my first PR 👀 

#### Additional comments
*Add any other context or screenshots about the feature request here.*

### PR DESCRIPTION
**Pull request by matsvanes**
Fixes #10725 .

Add `vmin` and `vmax` specification to `mne.Evoked.animate_topomap`

### CODE REVIEW

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -23,6 +23,7 @@ Current (1.2.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
+- Add `vmin` and `vmax` specification to `mne.Evoked.animate_topomap` (:gh:`10725` by :newcontrib:`Mats van Es`)
```
Comment (drammock) [inline doc/changes/latest.inc:26]:
```suggestion
- Add ``vmin`` and ``vmax`` parameters to :meth:`mne.Evoked.animate_topomap` (:gh:`10725` by :newcontrib:`Mats van Es`)
```

File: mne/evoked.py [hunk 2]
```diff
@@ -455,8 +455,9 @@ def plot_joint(self, times="peaks", title='', picks=None,
 
     @fill_doc
     def animate_topomap(self, ch_type=None, times=None, frame_rate=None,
-                        butterfly=False, blit=True, show=True, time_unit='s',
-                        sphere=None, *, image_interp=_INTERPOLATION_DEFAULT,
+                        vmin=None, vmax=None, butterfly=False, blit=True,
```
Comment (drammock) [inline mne/evoked.py:458]: typically we wouldn't add new params to the middle of the signature like this. After the `*` in the signature, they can go wherever makes sense, but before the `*` we implicitly guarantee a fixed order.

**Review by drammock — CHANGES_REQUESTED** [review]

File: mne/evoked.py (hunk 2 above)
Comment (matsvanes) [reply mne/evoked.py:458]: Ah I wasn't aware of this convention, but it makes sense. I'll change it accordingly

File: mne/evoked.py [hunk 3]
```diff
@@ -501,6 +500,8 @@ def animate_topomap(self, ch_type=None, times=None, frame_rate=None,
 
             .. versionadded:: 0.22
         %(verbose)s
+            .. versionadded:: 1.1.0
+        %(vmin_vmax_topomap)s
```
Comment (drammock) [inline mne/evoked.py:504]: in fact, my earlier comment was not quite correct (sorry!).  I forgot to mention that if there is a `verbose` param it should always come last.  So please move `vmin`/`vmax` to just before `verbose`.  Also the `.. versionadded::` bit should come *after* the parameter it's referring to, so switch the order of these two lines please (and also move them to before `verbose`).

File: mne/evoked.py [hunk 4]
```diff
@@ -455,10 +455,10 @@ def plot_joint(self, times="peaks", title='', picks=None,
...
     @fill_doc
     def animate_topomap(self, ch_type=None, times=None, frame_rate=None,
-                        vmin=None, vmax=None, butterfly=False, blit=True,
-                        show=True, time_unit='s', sphere=None, *,
-                        image_interp=_INTERPOLATION_DEFAULT,
-                        extrapolate=_EXTRAPOLATE_DEFAULT, verbose=None):
+                        butterfly=False, blit=True, show=True, time_unit='s',
+                        sphere=None, *, image_interp=_INTERPOLATION_DEFAULT,
+                        extrapolate=_EXTRAPOLATE_DEFAULT, verbose=None,
+                        vmin=None, vmax=None):
```
Comment (drammock) [inline mne/evoked.py:461]: here too, move `vmin` and `vmax` to come before `verbose`

**Review by drammock — COMMENTED** [review]
Note that I used the GitHub web UI to commit my earlier suggestion (about the changelog), so before making further changes you'll first need to pull down that new commit (`git fetch origin && git merge --ff-only origin/issue-10725`)

File: mne/evoked.py (hunk 4 above)
Comment (matsvanes) [reply mne/evoked.py]: Thanks for your guidance (and patience!)

File: doc/changes/latest.inc [hunk 5]
```diff
@@ -23,6 +23,7 @@ Current (1.2.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
+- Add ``vmin`` and ``vmax`` parameters to :meth:`mne.Evoked.animate_topomap` (:gh:`10725` by :newcontrib:`Mats van Es`)
```
Comment (drammock) [inline doc/changes/latest.inc:26]:
```suggestion
- Add ``vmin`` and ``vmax`` parameters to :meth:`mne.Evoked.animate_topomap` (:gh:`11073` by :newcontrib:`Mats van Es`)
```

(we put the PR number in changelogs, not the issue number)

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
LGTM!  Thanks @matsvanes; this will merge when CIs finish running

File: mne/evoked.py [hunk 6]
```diff
@@ -498,8 +499,12 @@ def animate_topomap(self, ch_type=None, times=None, frame_rate=None,
         %(extrapolate_topomap)s
 
             .. versionadded:: 0.22
+        %(vmin_vmax_topomap)s
+
+            .. versionadded:: 1.1.0
         %(verbose)s
 
+
```
Comment (drammock) [inline mne/evoked.py:507]:
```suggestion
```

**Pull request merged**